plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -236,6 +236,12 @@ func TestPodFitsResources(t *testing.T) {
 			test:     "opaque resource fits",
 		},
 		{
+			pod:      newResourceInitPod(newResourcePod(schedulercache.Resource{OpaqueIntResources: map[api.ResourceName]int64{opaqueResourceA: 1}})),
```

Comment:
This is not testing init containers. The first arg to `newResourceInitPod` is assigned to `pod.Spec.Containers` and the rest of the args are assigned to `pod.Spec.InitContainers`.  Should be changed as: `newResourceInitPod(newResourcePod(schedulercache.Resource{OpaqueIntResources: map[api.ResourceName]int64{opaqueResourceA: 1}}), schedulercache.Resource{OpaqueIntResources: map[api.ResourceName]int64{opaqueResourceA: 1}})` to test init containers.